mysql-test/suite/innodb/t/buf_pool_read_request.test

```diff
@@ -0,0 +1,16 @@
+--source include/have_innodb.inc
+--source include/have_sequence.inc
+--echo #
+--echo # MDEV-32390 innodb_buffer_pool_read_requests always 0
+--echo #
+create table t1(f1 int not null, f2 int not null)engine=innodb;
+insert into t1 select seq, seq from seq_1_to_16384;
+SELECT  VARIABLE_VALUE into @val
+FROM   INFORMATION_SCHEMA.GLOBAL_STATUS
+WHERE  VARIABLE_NAME LIKE 'innodb_buffer_pool_read_requests';
+select count(*) from t1;
+SELECT  VARIABLE_VALUE into @val2
+FROM   INFORMATION_SCHEMA.GLOBAL_STATUS
+WHERE  VARIABLE_NAME LIKE 'innodb_buffer_pool_read_requests';
+select @val2 - @val > 0;
```

Comment:
`VARIABLE_VALUE` is a string, not an integer. If we were comparing `@val2>@val`, we would need a cast to an integer, like it was done in 2f467de4c4851d2f9b0f3bec54f748d92349582a to avoid anomalies where `'10'>'9'` does not hold. With the subtraction, maybe there will be an implicit conversion from strings to integers.